website/pages/apply-theme.tsx

```diff
@@ -0,0 +1,239 @@
+import { useRouter } from "next/router"
+import {
+  Box,
+  Heading,
+  FormControl,
+  FormLabel,
+  Input,
+  FormHelperText,
+  Button,
+  HStack,
+  VStack,
+  Text,
+  Link,
+  extendTheme,
+  useClipboard,
+  useToast,
+} from "@chakra-ui/react"
+import PageContainer from "components/page-container"
+import Sidebar from "components/sidebar/sidebar"
+import { getRoutes } from "layouts/mdx"
+import _ from "lodash"
```

Comment:
please use `import { last } from 'lodash' `  or `import last from 'lodash.last'` as you'd otherwise import the entirety of lodash

Reply:
@ljosberinn Got it and I will update it. Forgot that we don't have `babel-plugin-lodash` setup here.

I noticed that there are some other places of `website` that `import _ from "lodash"` directly so I assume it's okay do to so. I could fix those places in another PR for sure :)

![JPEG-1-1](https://user-images.githubusercontent.com/922234/123547851-fd50e180-d794-11eb-96b0-ff0e5a32f275.jpg)

Reply:
Switched to `import last from 'lodash/last'`

Reply:
FYI, I've created a PR to take care of existing imports: https://github.com/chakra-ui/chakra-ui/pull/4318